mm2src/coins/utxo/utxo_block_header_storage/wasm/indexeddb_block_header_storage.rs

```diff
@@ -254,7 +257,7 @@ impl BlockHeaderStorageOps for IDBBlockHeadersStorage {
                 reader
                     .read()
                     .map_err(|e: serialization::Error| BlockHeaderStorageError::DecodeError {
-                        coin: ticker,
+                        coin: ticker.clone(),
```

Comment:
First time I noted that there is a redundant clone, but later I realized that it would be better to not to clone ticker at the beginning of function and just create a reference.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/4348c832447b39a94ec284bb7db38aadc02345dc/mm2src/coins/utxo/utxo_block_header_storage/wasm/indexeddb_block_header_storage.rs#L208-L212

Please create `let ticker = self.ticker.as_str();` at the beginning, bcz during the `get_last_block_header_with_non_max_bits` (as an example) you need to provide ticker as `&str` a lot and `String` type you need only in the couple places at the end.

Creating reference will let you to pass `&str` directly, so you dont need to type `&ticker` every time. Also you dont have to watch out for extra cloning, due to `ticker.to_string()` usage.

Could you change `let ticker = self.ticker.clone();` to `let ticker = self.ticker.as_str();` in other functions please?

Reply:
done